enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -2079,6 +2080,12 @@ func (c *FirecrackerContainer) Exec(ctx context.Context, cmd *repb.Command, stdi
 		if err := c.parseSegFault(result); err != nil {
 			log.CtxWarningf(ctx, "Segfault occurred during task execution (recycled=%v) : %s", c.recycled, err)
 		}
+		// Slow hrtimer interrupts can happen during periods of high contention
+		// and may help explain action failures - surface these in the executor
+		// logs.
+		if warning := c.parseSlowInterruptWarning(); warning != "" {
+			log.CtxWarningf(ctx, "Slow interrupt warning reported in kernel logs: %q", warning)
```

Comment:
nit: We could consolidate some of the logic in the 3 `parseX` functions. We translate the vmlog bytes buffer to a string 3 times in a row

Reply:
Factored out the logic for converting to string and replacing `\r\n` with `\n`